source/renderer/app/components/wallet/WalletSendForm.js

```diff
@@ -48,15 +47,11 @@ type Props = {
   currencyMaxIntegerDigits: number,
   currencyMaxFractionalDigits: number,
   currentNumberFormat: string,
-  calculateTransactionFee: (
-    address: string,
-    amount: number,
-    assets: AssetItems
-  ) => Promise<BigNumber>,
+  calculateTransactionFee: Function,
```

Comment:
is there a reason for making the type signature less specific?

Reply:
There is already a specification in the container, WalletSendPage. Having it also inside the component seems duplicated.